compiler/rustc_codegen_llvm/src/gotoc/function.rs

```diff
@@ -9,6 +9,28 @@ use rustc_middle::mir::{HasLocalDecls, Local};
 use rustc_middle::ty::{self, Instance, TyS};
 use tracing::{debug, warn};
 
+/// Utility to skip functions that can't currently be successfully codgenned.
```

Comment:
I think I agree with Ted that this makes more sense close to where it's used: whether we skip a function is a property of codegening the crate, not the function itself?

Reply:
Its actually used in function.rs. But I agree that that's maybe not where you would expect to find it.

Reply:
https://github.com/model-checking/rmc/issues/424